apps/desks.py

```diff
@@ -32,6 +32,7 @@
 
 
 logger = logging.getLogger(__name__)
+SIZE_MAX = 2147483647
```

Comment:
you can write 2_147_483_647 to make it more readable

Reply:
:+1: